File: Baekjoon/Python/Sorting/Q10814.py
import sys

n = int(sys.stdin.readline())
members = []

for _ in range(n):
    members.append(list(sys.stdin.readline().split()))
members.sort(key=lambda x: int(x[0]))

for i in range(n):
    print(members[i][0], members[i][1])

File: Baekjoon/Swift/BOJ_PS/BOJ_PS/Sorting/Q10814.swift
//
//  Q10814.swift
//  BOJ_PS
//
//  Created by 김태훈 on 2021/06/28.
//  나이순 정렬 / 정렬

typealias Person = (age: Int, name: String)

var person = [Person]()
for _ in 0..<Int(readLine()!)! {
    let input = readLine()!.split(separator: " ").map { String($0) }
    person.append((Int(input[0])!, input[1]))
}

person.sort { $0.age < $1.age }
person.forEach {
    print($0.age, $0.name)
}
